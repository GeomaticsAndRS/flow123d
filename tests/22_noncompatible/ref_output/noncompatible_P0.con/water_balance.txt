	"time"	"region"	"quantity [m(3)]"	"flux"	"flux_in"	"flux_out"	"mass"	"source"	"source_in"	"source_out"	"integrated_flux"	"integrated_source"	"error"
	0	"1d"	"water_volume"	0	0	0	0	0	0	0	0	0	0
	0	"2d"	"water_volume"	0	0	0	0	0	0	0	0	0	0
	0	".1d"	"water_volume"	28.987	0	28.987	0	0	0	0	0	0	0
	0	".2d_bottom_top"	"water_volume"	-45.1026	-45.1026	0	0	0	0	0	0	0	0
	0	"IMPLICIT BOUNDARY"	"water_volume"	1.68714e-08	-1.07885e-08	2.76598e-08	0	0	0	0	0	0	0
