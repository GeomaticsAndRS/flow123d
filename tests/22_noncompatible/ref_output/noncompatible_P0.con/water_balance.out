# ---------------------------------------- WATER BALANCE ----------------------------------------
# Time of computed water balance: 0.000000


# Boundary water balance:
# [boundary_id] [label]                  [total_balance][total_outflow][total_inflow][time]        
# -----------------------------------------------------------------------------------------------
  4             .1d                        29.1088       29.1088       0             0             
  3             .2d_bottom_top             -39.4171      0             -39.4171      0             
  -3            IMPLICIT BOUNDARY          1.44969e-08   2.4659e-08    -1.01621e-08  0             
# -----------------------------------------------------------------------------------------------
# total boundary balance                   -10.3083      29.1088       -39.4171      


# Source fluxes over material subdomains:
# [region_id]   [label]                  [total_balance]                            [time]        
# -----------------------------------------------------------------------------------------------
  1             1d                         0                                         0             
  2             2d                         0                                         0             
# -----------------------------------------------------------------------------------------------
# total sources balance                    0             


